### PR TITLE
create github-ci.yml

### PR DESCRIPTION
**Pull request by alibekashirali**

### CODE REVIEW

File: .github/workflows/github-ci.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        run: echo Hello, world!
+
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          echo Add other actions to build,
+          echo test, and deploy your project.
+  deploy:
+    name: Deploy
+    if: github.ref == 'refs/heads/main'
```
Comment (chapayevdauren) [inline .github/workflows/github-ci.yml:38]: @alibekashirali it should work for production branch not main

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+resource "aws_instance" "app_server" {
+  ami           = "ami-830c94e3"
+  instance_type = "t2.micro"
+
+  tags = {
+    Name = "ExampleAppServerInstance"
+  }
+}
+
```
Comment (chapayevdauren) [inline main.tf]: @alibekashirali please attach screenshots that resources are created in AWS

Comment (chapayevdauren) [reply main.tf]: please also push terraform.tfstate file

Comment (alibekashirali) [reply main.tf]: okey. Just uploaded these files

**Pull request closed without merging**